aten/src/ATen/native/native_functions.yaml

```diff
@@ -6382,11 +6382,14 @@
 
 - func: polygamma.out(int n, Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   device_check: NoCheck   # TensorIterator
+  structured: True
+  structured_inherits: TensorIteratorBase
   dispatch:
     CPU, CUDA: polygamma_out
 
 - func: polygamma(int n, Tensor self) -> Tensor
   device_check: NoCheck   # TensorIterator
+  structured_delegate: polygamma.out
   variants: method, function
   dispatch:
     CompositeExplicitAutograd: polygamma
```

Comment:
thanks! don't forget to delete this dispatch entry and the one below in `polygamma_` (needa increase the checks to barf on this)

Reply:
Ah. Thanks!!

Removed `dispatch` from `polygamma`. However can't remove it from `polygamma_`, more information in the following [comment](https://github.com/pytorch/pytorch/pull/51966#discussion_r611934666)

Reply:
Huh, and you're not allowed to remove it even if you keep the entry in `derivatives.yaml`?

Reply:
Yes.
For `polygamma_`, after removing the dispatch from `native_functions.yaml` and keeping the entry in `derivatives.yaml`,
while compiling, we get the following error.

```
AssertionError: There's a formula for polygamma_(or its functional variant) in derivatives.yaml. It's required to add a dispatch section for it with explicit supported backends e.g CPU/CUDA or CompositeExplicitAutograd in native_functions.yaml. Please see https://github.com/pytorch/pytorch/tree/master/aten/src/ATen/native#choosing-the-right-dispatch-keyword for instructions to choose the right dispatch keyword.
```